dotCMS/src/main/java/com/dotcms/rest/api/v1/index/ESIndexResource.java

```diff
@@ -67,392 +67,351 @@
 @Path("/v1/esindex")
 public class ESIndexResource {
 
+    private enum IndexAction{
+        ACTIVATE,
+        DEACTIVATE,
+        CLEAR,
+        OPEN,
+        CLOSE;
+        
+        public static IndexAction fromString(String actionStr) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Parameter 'actionStr' is not assigned and could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5632525)